ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -1,66 +1,56 @@
-import React from 'react';
-
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
 import pathOr from 'ramda/src/pathOr';
 import { OptimoBlock } from '#models/types/optimo';
-import headings from '#app/legacy/containers/Headings';
+import Heading from '#app/components/Heading';
 import Blocks from '#app/legacy/containers/Blocks';
-import paragraph from '#app/legacy/containers/Paragraph';
-import Text from '#app/components/Text';
-import { Post as PostType } from './types';
-
-// temporary solution to render LI/ OL blocks.
-const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
-  const listItems: (string | null)[] = blocks
-    .map(item =>
-      pathOr<string | null>(
-        null,
-        ['model', 'blocks', 0, 'model', 'text'],
-        item,
-      ),
-    )
-    .filter(text => typeof text === 'string');
-
-  if (listItems.length === 0) return null;
-
-  return (
-    <Text>
-      <ul>
-        {listItems.map((item, index) => (
-          // eslint-disable-next-line react/no-array-index-key
-          <li key={index}>{item}</li>
-        ))}
-      </ul>
-    </Text>
-  );
-};
+import Paragraph from '#app/legacy/containers/Paragraph';
+import UnorderedList from '#app/legacy/containers/BulletedList';
+import { Post as PostType, PostHeadingBlock } from './types';
+import styles from './styles';
 
-const PostHeadings = ({ headerBlocks }: { headerBlocks: OptimoBlock[] }) => {
-  const componentsToRender = {
-    headline: headings,
-    subheadline: headings,
-  };
+const PostHeadings = ({ headerBlock }: { headerBlock: PostHeadingBlock }) => {
+  const isHeadline = headerBlock.type === 'headline';
+  const headingText = headerBlock.model.blocks[0].model.blocks[0].model.text;
 
   return (
-    <Blocks blocks={headerBlocks} componentsToRender={componentsToRender} />
+    <Heading
+      level={3}
+      fontVariant={isHeadline ? 'sansBold' : 'sansRegular'}
+      size={isHeadline ? 'greatPrimer' : 'brevier'}
+      className="headingStyling"
+      css={isHeadline ? styles.postHeading : styles.postSubHeading}
+    >
+      {headingText}
+    </Heading>
   );
 };
 
 const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
   const componentsToRender = {
-    paragraph,
-    unorderedList,
-    orderedList: unorderedList,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is there anyway we can get rid of these warnings? Do we know what the props might be?

Would it also be possible to try and avoid prop spreading (as per [Component Standards Guidelines: Avoid JSX props spreading](https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/coding-standards-react-components--page#avoid-jsx-props-spreading) - although totally understandable if it's not possible or too involved, in which case, don't worry)

Reply:
Ah good point, I've had a go!